src/index.ts

```diff
@@ -92,7 +94,15 @@ logRejections();
 
 // get application log for debugging
 calypsoServer.get("/log", (req: express.Request, res: express.Response) => {
-  const appLog = fs.readFileSync("./logs/log.txt", "utf-8"); // todo change back from l
+  //const appLog = fs.readFileSync("./logs/log.txt", "utf-8"); // todo change back from l
+  const truncatedLog = execSync( 'tail -n 2000 ./logs/log.txt' );
```

Comment:
Can we use the async version? seems simple to wrap the `res.send` bit in a callback.